lang-ca.js

```diff
@@ -924,9 +930,11 @@ SnapTranslator.dict.ca = {
     'add comment':
         'afegeix un comentari',
     'undrop':
-        'recupera bloc',
+        'desfés',
     'undo the last\nblock drop\nin this pane':
-        'recupera l\'últim bloc\nque s\'hagi llençat',
+        'desfés l\'últim moviment\nde blocs',
+    'redrop':
+        'refés',
```

Comment:
Hi, @bromagosa! Adding new 'redrop', I changed 'undrop' consciously. @jmoenig has explained better than I can do in English :))
I've changed other strings with a free translation... because original English string maybe is not the right explanation now (features changing...). I prefer tot talk about this in #1551 because is more general about translations synchronization system. I'll try to make a specific proposal...